maistra/ci/pre-submit.sh

```diff
@@ -18,7 +18,58 @@ export ARCH
 # Fix path to the vendor deps
 sed -i "s|=/work/|=$(pwd)/|" maistra/bazelrc-vendor
 
-# Build
+# Build WASM extensions first
+bazel build \
+  --incompatible_linkopts_to_linklibs \
+  --config=release \
+  --config=${ARCH} \
+  --local_ram_resources=12288 \
+  --local_cpu_resources=4 \
+  --jobs=4 \
+  --disk_cache=/bazel-cache \
+  //extensions:stats.wasm \
+  2>&1 | grep -v -E "${OUTPUT_TO_IGNORE}"
+
+bazel build \
+  --incompatible_linkopts_to_linklibs \
```

Comment:
Can we simplify this? A lot of repeated code here and it needs to be extended as new extensions are added.

Reply:
don't spend time here. It's all experimental. Look at the branch.

Reply:
+1. I'll approve. Thanks.